src/interfaces.ts

```diff
@@ -84,25 +84,31 @@ export interface ICoreKit {
   /**
    * Login into the SDK in an implicit flow and initialize all relevant components.
    * @param loginParams - Parameters for Implicit Login.
-   * @param factorKey - A BN used for encrypting your Device/ Recovery TSS Key Share. Optional for new users, mandatory for existing users, if not provided we will try to fetch it from local storage.
-   * @returns A Web3 provider if we are not in redirect mode.
    */
-  login(loginParams: LoginParams, factorKey?: BN): Promise<SafeEventEmitterProvider | null>;
+  loginWithOauth(loginParams: OauthLoginParams): Promise<void>;
 
   /**
    * Login into the SDK using ID Token based login and initialize all relevant components.
    * @param idTokenLoginParams - Parameters with ID Token based Login.
-   * @param factorKey - A BN used for encrypting your Device/ Recovery TSS Key Share. Optional for new users, mandatory for existing users, if not provided we will try to fetch it from local storage.
-   * @returns A Web3 provider
    */
-  loginWithIdToken(idTokenLoginParams: IdTokenLoginParams, factorKey?: BN): Promise<SafeEventEmitterProvider | null>;
+  login(idTokenLoginParams: IdTokenLoginParams): Promise<void>;
```

Comment:
```suggestion
  loginWithIdToken(idTokenLoginParams: IdTokenLoginParams): Promise<void>;
```
For consistency with the naming above.